cppwinrt.props

```diff
@@ -39,7 +39,7 @@
     <CppWinRTBuildVersion Condition="'$(CppWinRTBuildVersion)'==''">2.3.4.5</CppWinRTBuildVersion>
     <CmakePlatform>$(Platform)</CmakePlatform>
     <CmakePlatform Condition="'$(Platform)'=='Win32'">x86</CmakePlatform>
-    <CmakeOutDir Condition="'$(CmakeOutDir)'==''">$(SolutionDir)_build\$(CmakePlatform)\$(Configuration)</CmakeOutDir>
+    <CmakeOutDir Condition="'$(CmakeOutDir)'==''">$(MSBuildThisFileDirectory)_build\$(CmakePlatform)\$(Configuration)</CmakeOutDir>
```

Comment:
How were you building cppwinrt that necessitated this change?

Reply:
Command-line MSBuild in the `fast_fwd`, `prebuild`, and `cppwinrt` directories. Attempting to build the top-level solution was not working, as the test projects were somehow incompatible with the version of cppwinrt in the repo at this time. I didn't know how (nor had the time) to fix it, so I went with the next best method.

Reply:
You need to build the cppwinrt project first - and then build a projection - and then build the rest. 

https://github.com/microsoft/cppwinrt/blob/master/build_test_all.cmd

Reply:
I'm happy to look at a solution that makes this simpler, but short of that we need to keep with the status quo.